misc/add_func.py

```diff
@@ -0,0 +1,160 @@
+            sys.exit(0)
+    except Exception :
+        print("enter the argument complete '-t' '-f' ")
+        sys.exit(0)
+    molecule_type = SequenceType.objects.filter(molecule_type = molecule2) #get the data from molecule_type field
+    a = molecule_type[0]
+    molecule_str = a.molecule_type
+    return molecule2,molecule_str
+
+def get_dataset(options):
+
+    dataset = options['type'][1].lower().capitalize()
+    if dataset =='Genome':
+        dataset = dataset + ' ' + options['type'][2].lower().capitalize()
+    elif dataset == 'Transcript':
+        pass
+    elif dataset == 'Protein':
+        pass
+    else:
+        print('enter the correct dataset type')
+        sys.exit(0)
+    dataset_type = SequenceType.objects.filter(dataset_type = dataset)
+    b = dataset_type[0]
+    dataset_str = str(b.dataset_type)
+    return dataset,dataset_str
+
+def get_type(dataset,molecule2,molecule_str,dataset_str): #get the sequence type from SequencType Table
+
+    if molecule2 != molecule_str :
+        print("something wrong in molecule")
+    elif dataset != dataset_str :
+        print("something wrong with dataset")
+    else:
+        try:
+            dataset_type = SequenceType.objects.filter(molecule_type = molecule2, dataset_type = dataset)
+            return dataset_type[0]
+        except IndexError:
+            print("there are no {molecule} - {dataset} combination in the database".format(molecule=molecule2.capitalize(),dataset=dataset_str))
+            sys.exit(0)
+def get_description(url1,wiki_url2):
+    try:
+        re1 = requests.get(url1)
+        data1 = re1.json()
+        try:
+            title = data1['query']['search'][0]['title']
+            url2 = wiki_url2 + title
+            re2 = requests.get(url2)
+            data2 = re2.json()
+            key = data1['query']['search'][0]['pageid']
+            key = str(key)
+            #print type(key)
+            description = data2['query']['pages'][key]['extract']
+            #print description
+            return description
+        except 	IndexError:
+            print("check your organism name again")
+            sys.exit(0)
+    except requests.exceptions.ConnectionError:
+        print("check your internet connection")
+        sys.exit(0)
+
+def get_taxid(id_baseurl,name):
+    try:
+        url = id_baseurl+ name
+        re = requests.get(url)
+        data = re.json()
+        tax_id = data['esearchresult']['idlist'][0]
+        tax_id = int(tax_id)
+        return tax_id
+    except IndexError:
+        print("make sure your name is completed and correct")
+        sys.exit(0)
+
+def delete_org(name):
+    #if options["organism"]:
+    #for organism in options["organism"]:
+    #organism = options["organism"][0].lower().capitalize() + " " + options["organism"][1].lower()
+    Organism.objects.filter(display_name = name).delete()
+    return ("remove %s in database"%name)
+'''
```

Comment:
Don't include these commented out lines. It seems to me that these lines are for initial development, right ?